modules/tinymce/src/plugins/image/test/ts/browser/DialogTest.ts

```diff
@@ -1,143 +1,142 @@
-import { FocusTools, Keyboard, Keys, Log, Pipeline } from '@ephox/agar';
-import { UnitTest } from '@ephox/bedrock-client';
+import { FocusTools, Keyboard, Keys } from '@ephox/agar';
+import { describe, it } from '@ephox/bedrock-client';
 import { Fun } from '@ephox/katamari';
-import { TinyApis, TinyLoader } from '@ephox/mcagar';
-import { SugarElement } from '@ephox/sugar';
+import { TinyHooks } from '@ephox/mcagar';
+import { SugarDocument } from '@ephox/sugar';
 
+import Editor from 'tinymce/core/api/Editor';
 import Plugin from 'tinymce/plugins/image/Plugin';
-import SilverTheme from 'tinymce/themes/silver/Theme';
+import Theme from 'tinymce/themes/silver/Theme';
 
-UnitTest.asynctest('browser.tinymce.plugins.image.DialogTest', (success, failure) => {
-
-  SilverTheme();
-  Plugin();
-
-  TinyLoader.setupLight((editor, onSuccess, onFailure) => {
-    const api = TinyApis(editor);
-    const doc = SugarElement.fromDom(document);
+describe('browser.tinymce.plugins.image.DialogTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    plugins: 'image',
+    toolbar: 'image',
+    indent: false,
+    base_url: '/project/tinymce/js/tinymce'
+  }, [ Plugin, Theme ]);
 
-    const sPressTab = Keyboard.sKeydown(doc, Keys.tab(), {});
-    const sPressEsc = Keyboard.sKeydown(doc, Keys.escape(), {});
-    const sPressDown = Keyboard.sKeydown(doc, Keys.down(), {});
+  const pressTab = () => Keyboard.activeKeydown(SugarDocument.getDocument(), Keys.tab(), { });
```

Comment:
I kinda like these... I wonder if we should use this style across the board move them into some test helper? Not for now, maybe something to consider?

Reply:
We discussed this in the call today and we said we're going to look at adding helpers to McAgar instead, as that'll still keep it short and reasonably clear. Will also prevent any confusion from having some use this and others not. eg:
```
TinyContentActions.keydown(editor, Keys.enter());
TinyUiActions.keydown(editor, Keys.enter());
```

Reply:
Sounds good